### DISCUSSION TITLE
404 Not Found when execute the code

### DISCUSSION THREAD

**Discussion by AnouarGanfoud** [executablebooks/meta#178] [Q&A]
I am trying to make my article executable, I followed the documentation and I configure my project, but when I try to execute using Binder, I am getting 404 not found error. I doubt that the problem is related to **path_to_book** in **_config.yml** which is on created on this way:
```
# Book settings
title: My sample book
author: The Jupyter Book Community
logo: logo.png

latex:
  latex_documents:
    targetname: book.tex

repository:
  url                       : https://github.com/AnouarGanfoud/Gentsch.git  # Online location of your book
  branch                    : master  # Which branch of the repository should be used when creating links (optional)	

launch_buttons:
  thebe                  : true
  binderhub_url: "https://mybinder.org"
```

**Comment by choldgraf** [comment]
can you post the URL that the Binder link contains? That will make it easier to debug, thanks

**Reply by AnouarGanfoud** [reply]
Hello Chris,
Thanks for trying to help me. This is the URL: https://hub.gke2.mybinder.org/user/anouarganfoud-gentsch-iw9629kq/tree/Gentsch.ipynb

**Reply by AnouarGanfoud** [reply]
@choldgraf is there any solution for that ? thanks in advance

**Reply by choldgraf** [reply]
hey @AnouarGanfoud - that URL is for an already-running Binder, but it is not sharable with others. Please share the binder link that is meant to be shared.

**Reply by choldgraf** [reply]
although looking at your link, it looks like your book is inside an `MA-jupyter` folder, while the link you shared does not include that folder. This is what `path_to_book` is for, so try setting it to `MA-jupyter`

**Reply by AnouarGanfoud** [reply]
@choldgraf  Many thanks, this was very helpful, I was using `public` in` path_to_book`, no the url is correct. Just getting a small problem that there is no R related kernel for my code
